examples/README

```diff
@@ -9,4 +9,4 @@ To run the demo:
       this means we are starting 3 EOS instances with 6 topic partitions and 50000 pre-populated records.
    5. Some notes for exactly once demo:
       5.1. The Kafka server has to be on broker version 2.5 or higher.
-      5.2. You could also use Intellij to run the example directly by configuring parameters as "Program arguments"
+      5.2. You could also use IntelliJ IDEA to run the example directly by configuring parameters as "Program arguments"
```

Comment:
Thanks, this change makes sense. Perhaps we want to link this keyword to Intellij's website? That would make it more explicit which Intellij are we talking about here.

Reply:
_@divijvaidya_ , Thanks your review. Therefore, I think it should be changed.

Reply:
ah, let me clarify further. I was wondering if we want to hyperlink this keyword to the website? Your change in that situation will look like:
```
[IntelliJ IDEA](https://www.jetbrains.com/idea/)
```

Reply:
_@divijvaidya_ , Thanks for the suggestion, I think this will work.

Reply:
ok, sounds good.